assets/javascripts/kubernetes-topology-graph.js

```diff
@@ -0,0 +1 @@
+../../dist/topology-graph.js
```

Comment:
Have you done this before?  Do symlinks work properly with the asset pipeline?  Just curious as I've never tried it myself.